config/locales/en/acts_as_span.yml

```
@@ -4,7 +4,7 @@ en:
       messages:
         end_date_propagator:
           propagation_failure: "%{parent} could not propagate
-           %{end_date_field_name} to %{child}:\n%{reason}"
+           %{end_date_field_name} to %{child}:\n%{reason}. Child ID: {child_id} "
```

```suggestion
           %{end_date_field_name} to %{child}:\n%{reason} (ID: {child_id}). "
```

"Child" is a term used internally, and is loosely defined ("the subject of a has-many relationship with a dependent destroy or delete action"). There could easily be cases where a record might not seem like a child but is now called one. Since we have already named the child record (the `child` key), we don't need to give it a nickname here.

---

(this suggestion was on an old commit, the syntax isn't right here) but the point remains!